Formula/julia.rb

```diff
@@ -80,8 +80,8 @@ def install
       LIBLAPACKNAME=libopenblas
       USE_BLAS64=0
       PYTHON=python3
-      MACOSX_VERSION_MIN=#{MacOS.version}
     ]
+    on_macos { args << "MACOSX_VERSION_MIN=#{MacOS.version}" }
```

Comment:
Probably a no-op change as I think Julia always checks if on Darwin before using this input.
Though, I don't think it hurts and it allows PR to attempt bottle.